tests/gdx-tests/src/com/badlogic/gdx/tests/extensions/ControllersTest.java

```diff
@@ -1,184 +1,283 @@
-/*******************************************************************************
```

Comment:
@intrigus Add [?w=1](https://github.com/libgdx/libgdx/pull/3458/files?w=1) to the url (someone at github should make that the default.